pkgs/applications/blockchains/nearcore/default.nix

```diff
@@ -1,32 +1,46 @@
 { rustPlatform, lib, fetchFromGitHub
-, zlib, elfutils, openssl
-, cmake, python3, pkg-config, protobuf, perl, llvmPackages
+, zlib, openssl
+, pkg-config, protobuf, llvmPackages
 }:
 rustPlatform.buildRustPackage rec {
-  #https://github.com/near/nearcore
   pname = "nearcore";
-  version = "1.25.0";
+  version = "1.26.0-rc.3";
+
+  # https://github.com/near/nearcore/tags
   src = fetchFromGitHub {
     owner = "near";
     repo = "nearcore";
     # there is also a branch for this version number, so we need to be explicit
     rev = "refs/tags/${version}";
-    sha256 = "sha256-7hiBqJLGIf+kNKJvMQ7KtGZm/SWLY3pT7YDlwbm3HDM=";
+    sha256 = "sha256-MjeCGTS9UPCfk4Wyj1yx0FnsVk7o8b/Ahf9Ex+7MUgw=";
   };
 
-  cargoSha256 = "sha256-EGv4CibSHL9oTAdWK7d/SOzZWPcEB16hTWlWHjKU4wc=";
+  cargoSha256 = "sha256-nadJn8irq7ZsXFXaYIy2ub0ZdDKhfFBaOaH/DOZTUVk=";
+
+  postPatch = ''
+    substituteInPlace neard/src/main.rs \
+      --replace 'git_version!(fallback = "unknown")' '"nix:${version}"' \
```

Comment:
One side note though: I'm not sure if you want to do this in official nixpkgs, might be better idea to include .git directory and let build script do the rest?

Reply:
Having `.git` introduces a lot of headaches because it makes the whole thing non-reproducible and leads to breaking checksums when fetching a repo.